kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java

```diff
@@ -17,13 +17,17 @@
 package io.fabric8.kubernetes.client;
 
 import io.fabric8.kubernetes.client.utils.Serialization;
+import io.fabric8.openshift.clnt.v3_1.DefaultOpenShiftClient;
```

Comment:
You need to fix this import @lordofthejars

Reply:
Done